pycbc/events/stat.py

```diff
@@ -567,6 +568,109 @@ def single(self, trigs):
                            dtype=numpy.float32), ndmin=1, copy=False)
 
 
+class CoincRateCalcStatistic(NewSNRSGStatistic):
+
+    """Detection statistic using an exponential falloff noise model.
+    Statistic calculates the log noise coinc rate for each
+    template over single-ifo newsnr values.
+    """
+
+    def __init__(self, files):
+        if not len(files):
+            raise RuntimeError("Can't find any statistic files !")
+        NewSNRSGStatistic.__init__(self, files)
+        # the stat file attributes are hard-coded as '%{ifo}-fit_coeffs'
+        parsed_attrs = [f.split('-') for f in self.files.keys()]
+        self.ifos = [at[0] for at in parsed_attrs if
+                     (len(at) == 2 and at[1] == 'fit_coeffs')]
+        if not len(self.ifos):
+            raise RuntimeError("None of the statistic files has the required "
+                               "attribute called {ifo}-fit_coeffs !")
+        self.fits_by_tid = {}
+        self.alphamax = {}
+        for i in self.ifos:
+            self.fits_by_tid[i] = self.assign_fits(i)
+            self.get_ref_vals(i)
+
+        self.get_newsnr = ranking.get_newsnr
```

Comment:
as far as I can see this will end up using newsnr as the single-ifo stat, not newsnr_sgveto?